### PR TITLE
feat: add logging config

### PR DESCRIPTION
**Pull request by natygyoon**
Our public FastAPI based services should be configured with a reasonable logging config (i.e., unstructured at info , others at warning).

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
-unstructured[local-inference]>=0.4.4
+unstructured>=0.4.4
```
Comment (qued) [inline requirements/base.in:1]: Why this change?

**Review by cragwolfe — APPROVED** [review]
